src/helpers/user.js

```diff
@@ -29,3 +31,16 @@ export const getUserId = () => {
   }
   throw new Error("user is only to be called from renderer");
 };
+
+export const shouldUpdateYet = (version: string, remoteConfig: any) => {
+  const userId = getUserId();
+  const generateNumber = seedrandom(`${userId}-${version}`);
```

Comment:
`Prando` is the lib we use on live-common that is less problematic, especially on diff JS env (seedrandom does not run on RN for instance. not that we would use here but i kinda feel this code could belong to https://github.com/LedgerHQ/ledger-live-common/blob/master/src/user.js which centralize the "derives from user id" logic)

Reply:
I replaced seedrandom with Prando